3_visualize/src/plot_gradient.R

```diff
@@ -63,16 +67,16 @@ plot_gradient <- function(ensemble_data, site_info, date_start, days_shown, out_
               alpha = 1,
               color = NA) +
     # observed temperature
-    geom_point(data = temp_obs, 
-               aes(
-                 x = time,
-                 y = obs_max_temp_f,
-                 color = ifelse(stat(y) > c_to_f(threshold), NA, stat(y))
-               ),
-               shape = 21,
-               stroke = 1,
-               size = 1.25, 
-               fill = "white") +
+    # geom_point(data = temp_obs, 
+    #            aes(
+    #              x = time,
+    #              y = obs_max_temp_f,
+    #              color = ifelse(stat(y) > c_to_f(threshold), NA, stat(y))
+    #            ),
+    #            shape = 21,
+    #            stroke = 1,
+    #            size = 1.25, 
+    #            fill = "white") +
```

Comment:
I think commenting this out is fine for now. In the long term we may want to add an additional target that would add the observed data when they are available.

Reply:
Sounds good.